### ISSUE TITLE
Transaction Names are not as documentation states

### ISSUE THREAD

**Issue by seanarmstrong87** [getsentry/sentry-dotnet#1467]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
Sentry.AspNetCore 3.13 on a dotnetcoreapp2.2 project.

### Steps to Reproduce

Basic setup as per documentation. UseSentry() in Program and 

In Startup.cs :
```
   public void Configure(
            IApplicationBuilder app,
            IHostingEnvironment env, 
            IApiVersionDescriptionProvider provider,
            IOptions<DocumentationOptions> documentationOptions)
        {
            app.UseEndpointRouting();
            
            app.UseSentryTracing();
            
            app.UseForwardedHeaders();
...
```

### Expected Result

As per this [page](https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/performance/instrumentation/automatic-instrumentation) I expect the transaction names in Sentry to look like  `GET /companies/{id}` 

### Actual Result
The transaction names look like `GET CompaniesV3.GetCompaniesAtAddress` which is the ControllerName.Method. How do I change the transaction name?

**Comment by lucas-zimerman** [comment]
Hi,
For ASP.NET Core 3.X and newer you'll get the `GET /person/{id}:` layout, but older .NET Core versions will use another naming convention: `GET controller.action.area`

**Comment by bruno-garcia** [comment]
There's a way to configure transaction names but it sounds like you'll get the value you expect by moving to a supported .NET version

**Comment by SimonCropp** [comment]
closing this as wont fix as 2.2 is out of support
